src/atomist.config.ts

```diff
@@ -26,8 +26,8 @@ export const configuration: Configuration = {
     name: pj.name,
     version: pj.version,
     teamIds: [
-         // "T1JVCMVH7",
-       "T5964N9B7",    // spring-team
+          "T1JVCMVH7",
+       //"T5964N9B7",    // spring-team
```

Comment:
Do we want this to be the default?

Reply:
nah. keeps moving. I'll make it an env var in a separate PR

Reply:
I've done this